app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,115 @@
+:root {
+  --blue: #3778c2;
+  --green: #5fb523;
+  --ash: #434b54;
+  --white: #fff;
```

Comment:
- [ ] just as previous reviewer mentioned kindly match the color of your app to the given design, currently your text is blue while it is somewhat gray In the design :+1: 
your design :point_down: 
![Screenshot from 2023-11-03 15-00-01](https://github.com/HafizMassamTabraizKhan/expense-tracker/assets/66097371/02f3da4a-f1a2-403d-8b6f-45bcdde2de83)
given design :point_down: 
![Screenshot from 2023-05-06 13-54-49](https://github.com/HafizMassamTabraizKhan/expense-tracker/assets/66097371/0c2e5f87-2d24-4f7a-8111-46a4c654dc44)